packages/builders/src/components/button/URLButton.ts

```diff
@@ -0,0 +1,34 @@
+import {
+	ButtonStyle,
+	ComponentType,
+	type APIButtonComponent,
+	type APIButtonComponentWithURL,
+} from 'discord-api-types/v10';
+import { Mixin } from 'ts-mixer';
+import { ButtonBuilder } from './Button.js';
+import { URLOrCustomIdButtonMixin } from './mixins/URLOrCustomIdButtonMixin.js';
+
+/**
+ * A builder that creates API-compatible JSON data for buttons with URLs.
+ */
+export class URLButtonBuilder extends Mixin(ButtonBuilder<APIButtonComponentWithURL>, URLOrCustomIdButtonMixin) {
```

Comment:
I think this should be called `LinkButtonBuilder` to match the style name. (Description, filename, and methods for adding this builder need to be updated)